### ISSUE TITLE
Landscape layout

### ISSUE THREAD

**Issue by rugheid** [richardtop/CalendarKit#22]
The layout falls apart in landscape mode.

**Comment by richardtop** [comment]
It's a known issue, see [roadmap](https://github.com/richardtop/CalendarKit#roadmap).
Two options are possible:
Either fix and use current layout in landscape (which is inconsistent with Apple's calendar
or re-write calendar to match Apple style.
I'll mark your task as a feature request for now.

**Comment by rugheid** [comment]
Ok, perfect. I would vote for the Apple compatible option, since there are currently no swift libraries that support a week-view.

**Comment by richardtop** [comment]
Agree, but actually, it is possible to implement both with option 1 as transitory.
At some stage it was working:
![image](https://cloud.githubusercontent.com/assets/8013017/23266474/f47126ac-f9f8-11e6-8c9d-4e4297067e5c.png)

**Comment by rugheid** [comment]
You're right, fixing the layout should be manageable in a shorter timeframe. The full week view would be a nice improvement for the future.

**Comment by niklasgrewe** [comment]
@richardtop any news on this topic. A week view would be awesome, especially for iPads. I think you probably don't have the time to implement this feature.... so could you give me an idea how to implement something like that? I am just starting to learn Swift and UIKit 😅

**Comment by richardtop** [comment]
Yes, I'm interested in this feature implemented.
To get it done, it would require creating a custom view/controller at the level of `TimelinePagerView`: https://github.com/richardtop/CalendarKit/blob/master/Source/Timeline/TimelinePagerView.swift

in that controller, you could use 5/7 components of `TimelineContainer` to display each column.
In addition to that, the new `WeekTimelinePagerView` will need to handle drag&drop & editing, which complicates things more.
But that's how you could code this functionality.

**Comment by richardtop** [comment]
![Simulator Screenshot - iPhone 15 - 2024-06-11 at 21 00 44](https://github.com/richardtop/CalendarKit/assets/8013017/9e86cdec-0108-4986-bd9a-50b951d291c7)

**Comment by richardtop** [comment]
![Simulator Screenshot - iPhone 15 - 2024-06-11 at 21 01 08](https://github.com/richardtop/CalendarKit/assets/8013017/97c8575e-31be-4d5d-8c5d-4da69ced5759)
